recipe/meta.yaml

```diff
@@ -18,6 +18,7 @@ build:
   script:
     - export MYPY_USE_MYPYC=1  # [not (win or (osx and arm64))]
     - set MYPY_USE_MYPYC=1     # [win]
+    - export MYPYC_OPT_LEVEL=2  # [aarch64 and linux]
```

Comment:
If this ends up working for py3.10, can that be restricted to only ` [aarch64 and linux and py=310]`?

Reply:
I'm just experimenting at the moment. Will make restrictions later.

Reply:
:+1:, just checking

I'm the maintainer for Debian's package of `mypy`: we also had to play around with the settings to get it to self-compile everywhere

We also run more build-time tests that this recipe currently does; feel free to steal from our scripts

https://salsa.debian.org/med-team/mypy/-/blob/master/debian/rules#L17
https://salsa.debian.org/med-team/mypy/-/blob/master/debian/changelog
https://salsa.debian.org/med-team/mypy/-/blob/master/debian/tests/run-unit-test ("in situ" tests that are run via ci.debian.net on installed packages)

Reply:
https://salsa.debian.org/med-team/mypy/-/blob/master/debian/patches/verbose may also be of interest

Reply:
Reported the issue upstream: https://github.com/python/mypy/issues/11507

Reply:
Thank you for sharing the links, I'll open an issue to add more tests in the recipe.